Asking for screenshot (#7382)
ankidroid/Anki-Android: .github/pull_request_template.md

- [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code

> - [ ] You provided screenshot of changes visible by the user (in particular, we need screenshot of all new strings ressources)

Reviewer: ```suggestion
- [ ] You have provided screenshots of all interface changes visible to the user (including screenshots of any new/changed string resources)
```

> - [ ] You provided screenshot of changes visible by the user (in particular, we need screenshot of all new strings ressources)

Reviewer 2: Well, that's just like a challenge. I can propose wording changes all day long hahaha

```suggestion
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
```
